src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1684,6 +1684,16 @@ public static function unsetConnectionResolver()
         static::$resolver = null;
     }
 
+    /**
+     * Get the table name for the model.
+     *
+     * @return string
+     */
+    public static function getTableName()
+    {
+        return (new static)->getTable();
```

Comment:
Seems like changing getTable to be static instead would be the better solution in 9.x.